src/website/src/utils/correctSvgPath.js

```diff
@@ -7,7 +8,11 @@ import iconPath from '@ecl/resources-ec-icons/dist/sprites/icons.svg';
 const correctSvgPath = (data) => {
   Object.keys(data).forEach((prop) => {
     if (typeof data[prop] === 'string' && data[prop].includes('.svg')) {
-      data[prop] = iconPath;
+      if (data[prop].includes('social')) {
```

Comment:
I had to do the same for the social-media-share, i wrote this as a ternart operator, though:
`data[prop] = data[prop].includes('social') ? iconSocialPath : iconPath`
if you like it, change it like this, otherwise it's fine as you wrote it.